cartographer/mapping_2d/local_trajectory_builder.cc

```diff
@@ -48,7 +48,7 @@ sensor::RangeData LocalTrajectoryBuilder::TransformAndFilterRangeData(
       sensor::VoxelFiltered(cropped.misses, options_.voxel_filter_size())};
 }
 
-void LocalTrajectoryBuilder::ScanMatch(
+bool LocalTrajectoryBuilder::ScanMatch(
```

Comment:
Please return `unique_ptr<transform::Rigid2d>`, it is more consistent with FastCorrelativeScanMatcher.

Reply:
Done.